src/nos/core/orchestrator.py

```diff
@@ -254,8 +254,19 @@ async def _iva_unverified(
     ):
         """Send notifications for IVAs set to 'unverified'.
 
-        This happens when the user exceeds the allotted time to submit their IVA code.
+        This happens when the user exceeds the allotted time to submit their IVA code or
```

Comment:
"this *usually* happens..."

Reply:
Changed in [17e0b05](https://github.com/ghga-de/notification-orchestration-service/pull/14/commits/17e0b05f6ff4e8598171a8e2fc01401c3046073c)